github/PullRequest.py

```diff
@@ -890,6 +890,21 @@ def remove_from_assignees(self, *assignees):
         )
         self._useAttributes(data)
 
+    def update_branch(self, expected_head_sha):
+        """
+        :calls `PUT /repos/:owner/:repo/pulls/:pull_number/update-branch <https://developer.github.com/v3/pulls>`_
+        :param expected_head_sha: string
+        :rtype: None
+        """
+        assert isinstance(expected_head_sha, str)
```

Comment:
We can't do this yet -- it may be unicode for Python 2 (for just a little while longer!), follow the pattern used elsewhere in this file for six.text_type

Reply:
:) sure thing - only 19 days to go 👍 

- done